src/inmanta/server/server.py

```diff
@@ -173,7 +175,17 @@ def _ensure_directory_exist(directory: str, *subdirs: str) -> str:
         slices = []
         extension_names = set()
         for slice_name, slice in self._server.get_slices().items():
-            slices.append(SliceStatus(name=slice_name, status=await slice.get_status()))
+            try:
+                slices.append(SliceStatus(name=slice_name, status=await slice.get_status()))
+            except (PostgresError, InterfaceError) as e:
```

Comment:
The stacktrace in the ticket shows an `OSError`, but this one is not caught here.